components/Loading.tsx

```diff
@@ -1,12 +1,35 @@
 import { Text, View } from './Themed';
 import TopBar from './TopBar';
-import React from 'react';
-import { StyleSheet } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import { Button, StyleSheet } from 'react-native';
+import Logout from '../lib/Logout';
+import { useNavigation } from '@react-navigation/native';
+import getLoggedInUser from '../lib/getLoggedInUser';
+import User from '../types/User';
 
 export default function Loading() {
+  const navigation = useNavigation();
+
+  const [user, setUser] = useState<User>();
+  const [refresh, setRefresh] = useState(Math.random());
+
+  async function CustomLogout() {
+    setRefresh(Math.random());
+
+    await Logout(navigation.navigate);
+  }
+
+  useEffect(() => {}, [refresh]);
```

Comment:
What's the use of this useEffect? To me it looks like there is nothing happening in the effect(empty function)

Reply:
is funny